src/components/counter/counter.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+
+function Counter({ initialValue }) {
+  const [value, setValue] = useState(initialValue || 0);
+
+  const materialButtonStyles = {
+    backgroundColor: '#2196F3',
+    color: 'white',
+    padding: '10px 20px',
+    borderRadius: '4px',
+    border: 'none',
+    cursor: 'pointer',
+    transition: 'background-color 0.3s',
+    boxShadow: '0 2px 4px rgba(0, 0, 0, 0.2)',
+    fontWeight: 'bold',
+    fontSize: '16px',
+    outline: 'none',
+    textDecoration: 'none',
+    display: 'inline-block',
+    textAlign: 'center',
+  };
+
+  const divStyle = {
+    width: '100%',
+    margin: '1% auto',
+    backgroundColor: '#f8f8f8',
+    padding: '20px',
+    borderRadius: '5px',
+    boxShadow: '0 2px 4px rgba(0, 0, 0, 0.1)',
+    display: 'flex',
+    alignItems: 'center',
+    gap: '10px',
+    flexDirection: 'column'
+  };
+
+  const divStyleRow = {
+    width: '80%',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: '10px',
+  };
+
+
+  const decrementValue = () => { setValue(value - 1) };
```

Comment:
nit: for some state changes, due to the async nature of useState, you could use the previous value using this version of the useState:
```
const decrementValue = () => setValue((previousValue) => previousValue - 1);
```